packages/forms/src/UIForm/fields/CheckBox/displayMode/TextMode.component.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '@talend/react-components/lib/Icon';
+import FieldTemplate from '../../FieldTemplate/displayMode/TextMode.component';
+
+import theme from './TextMode.scss';
+
+export default function TextModeCheckBox({ id, schema, value }) {
+	return (
+		<FieldTemplate id={id} label={schema.title || value} className={theme.checkbox}>
+			<Icon
+				aria-label={value}
+				aria-hidden={false}
+				name={value ? 'talend-check' : 'talend-cross'}
+				className={classNames(theme.icon, { [theme.cross]: !value })}
+			/>
+		</FieldTemplate>
+	);
+}
+
+if (process.env.NODE_ENV !== 'production') {
+	TextModeCheckBox.propTypes = {
+		id: PropTypes.string,
+		schema: PropTypes.shape({
+			title: PropTypes.string,
+		}),
+		value: PropTypes.bool,
```

Comment:
i think all those should be required, especially schema to avoid runtime error on schema.title

Reply:
Value no, because it can be unset, considered false by default (checkbox widget behavior)
Schema: I'll make it required